pkg/ottl/contexts/internal/ctxprofilecommon/attributes.go

```diff
@@ -34,9 +34,18 @@ func AccessAttributes[K any](source attributeSource[K]) ottl.StandardGetSetter[K
 			dict, attributable := source(tCtx)
 			attributable.AttributeIndices().FromRaw([]int32{})
 			for k, v := range m.All() {
-				if err := pprofile.PutAttribute(dict.AttributeTable(), attributable, dict, k, v); err != nil {
+				kvu := pprofile.NewKeyValueAndUnit()
```

Comment:
Should we concerned about the `pprofile.NewKeyValueAndUnit` docs saying `This must be used only in testing code`?